### PR TITLE
[WIP] Add Bloomberg as user

### PR DESCRIPTION
**Pull request by mosabua**
Depends on #485

### CODE REVIEW

File: _data/users.yml [hunk 1]
```diff
@@ -122,6 +122,20 @@
   links:
     - urltext: View the Bazaar Technologies website
       url: https://www.bazaar-tech.com/
+- name: Bloomberg
+  highlight: false
+  anchor: bloomberg
+  logo: /assets/images/logos/bloomberg.png
+  logosmall: /assets/images/logos/bloomberg-small.png
+  testimonial: |
+    Trino is a critical piece of the data platform infrastructure at Bloomberg.
```
Comment (bitsondatadev) [inline _data/users.yml:131]: That's it? That's like saying food is a critical part of nutrition.

File: _data/users.yml [hunk 2]
```diff
@@ -122,6 +122,20 @@
...
+  links:
+    - urltext: View the Bloomberg open source website
+      url: https://www.adlibertas.com/
```
Comment (bitsondatadev) [inline _data/users.yml:134]: Fix this url

File: _data/users.yml [hunk 3]
```diff
@@ -122,6 +122,20 @@
   links:
     - urltext: View the Bazaar Technologies website
       url: https://www.bazaar-tech.com/
+- name: Bloomberg
+  highlight: false
```
Comment (bitsondatadev) [inline _data/users.yml:126]: I assume this keeps them at the forefront of the first page? Perhaps Bloomberg should be on this list me thinks.

**Review by bitsondatadev — CHANGES_REQUESTED** [review]
Few fixes/questions

File: _data/users.yml (hunk 1 above)
Comment (mosabua) [reply _data/users.yml:131]: Hahah .. of course thats not it.. just a placeholder while we work on the real thing.

File: _data/users.yml (hunk 3 above)
Comment (mosabua) [reply _data/users.yml:126]: They will show up as sponsor for. We will also see about users selection, but thats a separate effort.

**Comment by bitsondatadev** [comment]
Looks good

**Review by bitsondatadev — APPROVED** [review]
Looks good

**Comment by mosabua** [comment]
Uff .. this was WIP .. and should NOT have been merged.. I will undo.

**Pull request merged**